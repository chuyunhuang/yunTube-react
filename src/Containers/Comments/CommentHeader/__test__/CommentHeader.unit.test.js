import React from 'react';
import { shallow } from 'enzyme';
import { CommentHeader } from '../CommentHeader';

describe('CommentHeader', () => {
  test('renders CommentHeader with props.amountComment= null', () => {
    const wrapper = shallow(<CommentHeader />)
    expect(wrapper).toMatchSnapshot();
  });
  test('renders CommentHeader with props.amountComments= 123', () => {
    const wrapper = shallow(<CommentHeader amountComments={123} />)
    expect(wrapper).toMatchSnapshot();
  });
})