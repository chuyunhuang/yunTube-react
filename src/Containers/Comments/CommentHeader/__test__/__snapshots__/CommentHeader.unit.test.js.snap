// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CommentHeader renders CommentHeader 1`] = `
<div
  className="comments-header"
>
  <h4>
    Comments
  </h4>
  <Button
    as="button"
    basic={true}
    compact={true}
    icon={true}
    labelPosition="left"
  >
    <Icon
      as="i"
      name="align-left"
    />
    Sort By
  </Button>
</div>
`;

exports[`CommentHeader renders CommentHeader with props.amountComment= null 1`] = `
<div
  className="comments-header"
>
  <h4>
     Comments
  </h4>
  <Button
    as="button"
    basic={true}
    compact={true}
    icon={true}
    labelPosition="left"
  >
    <Icon
      as="i"
      name="align-left"
    />
    Sort By
  </Button>
</div>
`;

exports[`CommentHeader renders CommentHeader with props.amountComments= 123 1`] = `
<div
  className="comments-header"
>
  <h4>
    123
     Comments
  </h4>
  <Button
    as="button"
    basic={true}
    compact={true}
    icon={true}
    labelPosition="left"
  >
    <Icon
      as="i"
      name="align-left"
    />
    Sort By
  </Button>
</div>
`;
